src/Scientist/IObservationPublisher.cs

```diff
@@ -8,10 +8,10 @@ namespace GitHub
     public interface IObservationPublisher
```

Comment:
Given the change below, we should probably rename this interface to `IResultPublisher` right?